packages/desktop-client/src/components/autocomplete/CategoryAutocomplete.tsx

```diff
@@ -183,6 +199,34 @@ export function CategoryAutocomplete({
     [defaultCategoryGroups, categoryGroups, showSplitOption],
   );
 
+  const filterSuggestions = useCallback(
+    (
+      suggestions: CategoryAutocompleteItem[],
+      value: string,
+    ): CategoryAutocompleteItem[] => {
+      return suggestions
+        .filter(suggestion => {
+          const suggestionAsItem = suggestion as CategoryAutocompleteItem; // No need for casting to Suggestion here
+          return (
+            suggestionAsItem.id === 'split' ||
+            suggestionAsItem.group?.name
+              .toLowerCase()
+              .includes(value.toLowerCase()) ||
+            (suggestionAsItem.group?.name + ' ' + suggestionAsItem.name)
+              .toLowerCase()
+              .includes(value.toLowerCase()) ||
+            defaultFilterSuggestion(suggestionAsItem, value)
+          );
+        })
+        .toSorted(
```

Comment:
Ok, looks like `toSorted` isn't available in the version of TS (I assume?) we're using. Sorry about that. Lets bring back `.sort()`